.travis.yml

```diff
@@ -7,13 +7,21 @@ os:
   - linux
   - windows
   - osx
-sudo: false
 script:
-  - cargo build --verbose
   - cargo test --verbose
-notifications:
-  email:
-    on_success: never
-branches:
-  only:
-    - master
+  - cargo build --verbose
```

Comment:
Looking forward to merging your PR 👍 ! Just one quick question:
Why move `cargo build` below `cargo test`? Tests require a successful build. If the project has not been built, `cargo test` will build it, so building *after* tests have run successfully is a bit redundant.

Reply:
No very good reason, personally I would first execute the tests, and then build, but that's the complete reason. But we maybe should change the ```cargo build``` to ```--release```, and then there are different builds, which makes it irrelevant.

So, Should I change it, or make cargo build a release build?